stdlib/lang-parser/resources/expression/binaryExpression/output(AST)/excess_operator.txt

```diff
@@ -0,0 +1,170 @@
+function foo(){
+int a = b ++;
+}
+
+serialized json output:
+ERROR [sachini/parserModule] - 2:12 : missing unary expression
```

Comment:
These error msgs should be updated to reflect the module name change.

Reply:
I have updated the org-name with `ballerina-parser`, the Module name in the project is `parserModule` which is why it is kept as `parserModule`.

Reply:
Sorry, I meant the organization name change.